scripts/Course.lua

```diff
@@ -88,6 +88,12 @@ function Course:getFieldPolygon()
 	return self.fieldPolygon
 end
 
+--- Any vine nodes found on the field?
+function Course:hasVineNodesOnTheField()
```

Comment:
This really isn't the business of the Course class, there is some logic outside of it that needs it.